Source/ACE.Entity/BaseAceObject.cs

```diff
@@ -5,12 +5,60 @@
 
 namespace ACE.Entity
 {
-    [DbTable("base_ace_object")]
-    [DbGetList("base_ace_object", 15, "typeId")]
+    [DbTable("ace_object")]
+    [DbGetList("ace_object", 15, "typeId")]
     public class BaseAceObject
     {
-        [DbField("baseAceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
+        [DbField("AceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
         public virtual uint AceObjectId { get; set; }
+
+        [DbField("bitField", (int)MySqlDbType.UInt32)]
```

Comment:
Thanks for taking the time to review and good catch.   I will fix it.   As you can tell - I am not yet actually pulling the data or that would have blown up.   :)